src/app/checklists/items-list/item/item.component.html

```diff
@@ -39,7 +39,7 @@
       class="action-icon"
       matTooltip="Indent left"
       [attr.aria-label]="'Indent ' + it.prompt + ' left'"
-      [disabled]="it.indent === 0 || it.centered || it.type === itemType.ITEM_SPACE"
+      [disabled]="it.indent === 0 || it.centered"
```

Comment:
I see that you changed test-data.ts, but can you please verify that none of the formats has any issue with indented blank spaces? (e.g. do the text-based formats interpret it correctly on export/import?)

Reply:
Removed, see last comment in #195.